app/facades/rev_it_up_facade.rb

```diff
@@ -0,0 +1,27 @@
+class RevItUpFacade
```

Comment:
This facade looks great and is what Brian was showing us in the refactor so good job :D